README.md

```diff
@@ -8,47 +8,106 @@ Rust-Lightning
-changes to the API, so any developer using this API at this stage should be prepared
-to embrace that. The current state is sufficient for a developer or project to
-experiment with it. Recent increased contribution rate to the project is expected
-to lead to a high quality, stable, production-worthy implementation in 2020.
+improve. It is also anticipated that as developers begin using the API, the
+lessons from that will result in changes to the API, so any developer using this
+API at this stage should be prepared to embrace that. The current state is
+sufficient for a developer or project to experiment with it. Recent increased
+contribution rate to the project is expected to lead to a high quality, stable,
+production-worthy implementation in 2021.
 
 Communications for Rust-Lightning and Lightning Development Kit happens through
 [LDK slack](http://lightningdevkit.org/).
 
+About
+-----------
+LDK/Rust-Lightning is a generic library which allows you to build a lightning
+node without needing to worry about getting all of the lightning state machine
+and on-chain punishment code exactly correct. Note that Rust-Lightning isn't, in
+itself, a node. There are various working/in progress demos which could be used
+as a node today, but if you "just" want a generic lightning node, you're almost
+certainly better off with `c-lightning`/`lnd` - if, on the other hand, you want
+to integrate lightning with custom features such as your own chain sync, your
+own key mangement, your own hardware, your own data storage/backup logic, etc.,
+LDK is likely your only option.
+
+We are currently working on a demo node which fetches blockchain data and
+on-chain funds via bitcoin core RPC/REST. The individual pieces of that demo
+are/will be composable, so you can pick the off-the-shelf parts you want and
+replace the rest.
+
+In general, Rust-Lightning does not provide (but LDK has implementations of):
+* on-disk storage - you can store the channel state any way you want - whether
+  google drive/icloud, a local disk, any K-V store/database/a remote server, or
+  any combination of them - we provide a clean api that provides objects which
+  can be serialized into simple binary blobs, and stored in any way you wish.
+* blockchain data - we provide a simple `block_connected`/`block_disconnected`
+  API which you provide block headers and transaction information to. We also
+  provide an API for getting information about transactions we wish to be
+  informed of, which is compatible with Electrum server requests/neutrino
+  filtering/etc.
+* UTXO management - RL/LDK owns on-chain funds as long as they are claimable as
+  a part of a lightning output which can be contested - once a channel is closed
+  and all on-chain outputs are spendable only by the user, we provide users
+  notifications that a UTXO is "theirs" again and it is up to them to spend it
+  as they wish. Additionally, we do not provide a way to fund channels aside
```

Comment:
s/we do not provide a way to fund channels aside from a generic API/channel funding is accomplished with a generic API/